### ISSUE TITLE
I´m sorry. Something went wrong with processing. Please wait a few seconds and try again, or contact support@pieces.app

### ISSUE THREAD

**Issue by CPX10** [pieces-app/support#471]
### Software

Desktop Application

### Operating System / Platform

Windows

### Your Pieces OS Version

10.1.13

### Early Access Program

- [x] Yes, this is related to an Early Access Program feature.

### Kindly describe the bug and include as much detail as possible on what you were doing so we can reproduce the bug.

[com.pieces.pfd.zip](https://github.com/user-attachments/files/17891327/com.pieces.pfd.zip)
![Image](https://github.com/user-attachments/assets/c9a6f930-4d30-41ad-8d70-b81d9c3f7e6f)
![Image](https://github.com/user-attachments/assets/99d91958-8fb7-4f27-9efa-9d628127e90f)
![Image](https://github.com/user-attachments/assets/8d1c630b-816c-488d-87f9-202a86d63820)
![Image](https://github.com/user-attachments/assets/8e8278a0-eca0-4b4e-bcaa-a5b195da5562)

**Comment by CPX10** [comment]
I have this issue, that an error message always appears, and i dont know, how to solve it

**Comment by CPX10** [comment]
[logs.zip](https://github.com/user-attachments/files/17891347/logs.zip)

**Comment by CPX10** [comment]
![Image](https://github.com/user-attachments/assets/db113353-6ddc-4aa9-aaa9-011d71dfd596)

**Comment by el3shka** [comment]
Hello everyone!
I have the same thing.
I don't use VPN, it started just a day ago.

**Comment by CPX10** [comment]
> Hello everyone! I have the same thing. I don't use VPN, it started just a day ago.

I don´t use vpn bro
So whats the matter?

**Comment by el3shka** [comment]
> > Hello everyone! I have the same thing. I don't use VPN, it started just a day ago.
> 
> I don´t use vpn bro So whats the matter?
no-no, im say - i dont use vpn and pieces anyway dont work, thats i mean. 
tryna with VPN - dont work too

**Comment by CPX10** [comment]
> > > Hello everyone! I have the same thing. I don't use VPN, it started just a day ago.
> > 
> > 
> > I don´t use vpn bro So whats the matter?
> > no-no, im say - i dont use vpn and pieces anyway dont work, thats i mean.
> > tryna with VPN - dont work too

I talk to them on discord and reserve a meet to solve my issue, cause with copilot, claude, chat bison and gemini doesn´t work

**Comment by mark-at-pieces** [comment]
hey @CPX10 && @el3shka this looks to be an issue related to ratelimiting, that we introduce recently within PiecesOS. We can increase your limit if you could schedule a brief call w/ myself here https://calendar.app.google/dVfa5wshULymRQyp8

**Comment by mark-at-pieces** [comment]
for anyone else that is having the same issue please add some time on my calender and I will increase you rate limit as well as resolve and other issues that you may be experiencing!

**Comment by CPX10** [comment]
> hey [@CPX10](https://github.com/CPX10) && [@el3shka](https://github.com/el3shka) this looks to be an issue related to ratelimiting, that we introduce recently within PiecesOS. We can increase your limit if you could schedule a brief call w/ myself here https://calendar.app.google/dVfa5wshULymRQyp8

I already schedule a meet call for tomorrow 
![Image](https://github.com/user-attachments/assets/e01964be-0bd9-432b-bc5a-ca40fdbe73ec)

**Comment by hra42** [comment]
@mark-at-pieces do you consider showing an error message for this? So we can direct people in the right way. Currently it's impossible to see even from looking at the logs.

**Comment by BenBackk** [comment]
hi @mark-at-pieces, im experiencing the same issue. Is there a way we can fix this without booking a Google Meet session with you? Looking forward to your response

**Comment by mark-at-pieces** [comment]
@hra42 yeah, we will upgrade the message w/ some infrastructure upgrade here likely within the month

**Comment by mark-at-pieces** [comment]
> I'm having the same issue. Local is working ok but cloud based gives error.

If you wouldn't mind pasting this request in your browser 'http://localhost:1000/applications' while your PiecesOS is running that will enable me to have the proper information.(and then sending the response to my email)

My email is mark@pieces.app.

This will resolve your issue.

TLDR: we added in some rate limiting last week, in the short term we will be manually allowing users that get blocked.

In the longterm, we will unfortunately have to require authentication to utilize cloud processing.

**Comment by mark-at-pieces** [comment]
> > > I'm having the same issue. Local is working ok but cloud based gives error.
> > 
> > 
> > If you wouldn't mind pasting this request in your browser 'http://localhost:1000/applications' while your PiecesOS is running that will enable me to have the proper information.(and then sending the response to my email)
> > My email is [mark@pieces.app](mailto:mark@pieces.app).
> > This will resolve your issue.
> > TLDR: we added in some rate limiting last week, in the short term we will be manually allowing users that get blocked.
> > In the longterm, we will unfortunately have to require authentication to utilize cloud processing.
> 
> Responded, can you check it Mark?

absolutely, I responded over email a little bit ago lemme know if things are working properly for you.

We also also tracking something within this flow that might be causing issues for folks. However would be helpful to know if this is now working properly for you.

**Comment by mark-at-pieces** [comment]
Hi everyone!

The issue here should now be resolved.

Please give the copilot another shot, however the rate-limiting has not gone away, at least for the time being.

If you are experiencing the issue still please following steps:
1. Paste this request in your browser 'http://localhost:1000/applications' while your PiecesOS is running
2. then send the response to my email [mark@pieces.app](mailto:mark@pieces.app).

Thank you everyone for helping us improve the product!

**Comment by mark-at-pieces** [comment]
@hra42 The best thing to do would be to just follow the steps up above if people are still experiencing the same issues.

and then forward the response to my email.

Thank you for you help!

**Comment by mark-at-pieces** [comment]
@all Final update here:

1. we had a little bug server side, causing issues w/ the rate limiting algo that was over rate limiting users. (This is now resolved)

2. This means that now if you are getting this "I´m sorry. Something went wrong with processing. Please wait a few seconds and try again, or contact support@pieces.app", you are likely rate limited, and will need to follow these steps 

"
1. Paste this request in your browser 'http://localhost:1000/applications' while your PiecesOS is running
note: port is 1000 for windows/macos, and 5323 for linux
2. then send the response to my email [mark@pieces.app](mailto:mark@pieces.app).
"

Finally, If you are still having issues do not hesitate to reach out to my email, and we will get your issues resolved!

Thank you all for being patient with us!

**Comment by mark-at-pieces** [comment]
on a side note: would be awesome if people would retry and comment if this is now up and running for them, and also if not please comment here as well ping me via email with the response :)

**Comment by hra42** [comment]
@mark-at-pieces fyi: a few people wrote me on discord that their issue is now resolved, so it seems to work!

**Comment by BeZazz** [comment]
I am getting this issue. Not sure what to do, have tried to reach out via email, with no luck. So if anyone can help out, it would be appreciated.

**Comment by hra42** [comment]
> I am getting this issue. Not sure what to do, have tried to reach out via email, with no luck. So if anyone can help out, it would be appreciated.

Hey @BeZazz 

Please make sure you follow the steps above:

1. Paste this request in your browser 'http://localhost:1000/applications' while your PiecesOS is running
note: port is 1000 for windows/macos, and 5323 for linux
2. then send the response to my email mark@pieces.app.

If you have done so, please give @mark-at-pieces some time, so he can analyze and get back to you.

**Comment by mark-at-pieces** [comment]
> I am getting this issue. Not sure what to do, have tried to reach out via email, with no luck. So if anyone can help out, it would be appreciated.

hi @BeZazz I'm sorry I don't believe I received and email from you, or very well could have missed it on my-side.

Would you mind sending a follow up, I will get back with you as soon as possible.

**Comment by BeZazz** [comment]
On 06/12/2024 00:01, Mark Widman wrote:
>> I am getting this issue. Not sure what to do, have tried to reach
>> out via email, with no luck. So if anyone can help out, it would be
>> appreciated.
> 
> hi @BeZazz [1] I'm sorry I don't believe I received and email from
> you, or very well could have missed it on my-side.
> 
> Would you mind sending a follow up, I will get back with you as soon
> as possible.
> 
Here is the last one I sent I sent.

Hi Mark,
Sorry for all the emails, this will be my last. (getting differing 
advice of what to provide you)
Not trying to rush you but provide relent information.

This is in relation to the message.
I´m sorry. Something went wrong with processing. Please wait a few 
seconds and try again, or contact ***@***.***

http://localhost:5323/applications
{"schema":{"migration":0,"semantic":"MAJOR_0_MINOR_0_PATCH_1"},"iterable":[{"schema":{"migration":0,"semantic":"MAJOR_0_MINOR_0_PATCH_1"},"id":"193b632e-884f-4b7a-8346-f71dbe0ffa2e","name":"PIECES_FOR_DEVELOPERS","version":"unknown","platform":"LINUX","onboarded":true,"privacy":"OPEN","capabilities":"BLENDED","mechanism":"MANUAL","automaticUnload":false},{"schema":{"migration":0,"semantic":"MAJOR_0_MINOR_0_PATCH_1"},"id":"424ab5f8-b96c-4db5-b239-ea9133ae33aa","name":"VS_CODE","version":"0.0.1","platform":"LINUX","onboarded":false,"privacy":"OPEN","capabilities":"BLENDED","mechanism":"MANUAL","automaticUnload":false},{"schema":{"migration":0,"semantic":"MAJOR_0_MINOR_0_PATCH_1"},"id":"767db256-315e-4d69-90d2-c7adab7df070","name":"JETBRAINS","version":"10.0.0","platform":"LINUX","onboarded":false,"privacy":"OPEN","capabilities":"BLENDED","mechanism":"MANUAL","automaticUnload":false},{"schema":{"migration":0,"semantic":"MAJOR_0_MINOR_0_PATCH_1"},"id":"78c3de4e-17f0-4b60-b551-194df7e3854
2","name":"GOOGLE_CHROME_EXTENSION_MV3","version":"3.0.5","platform":"LINUX","onboarded":false,"privacy":"OPEN","capabilities":"BLENDED","mechanism":"MANUAL","automaticUnload":false},{"schema":{"migration":0,"semantic":"MAJOR_0_MINOR_0_PATCH_1"},"id":"c722bd3f-690e-4c82-93f6-3b1a7937ac80","name":"SUBLIME","version":"1.4.0","platform":"LINUX","onboarded":false,"privacy":"OPEN","capabilities":"BLENDED","mechanism":"MANUAL","automaticUnload":false},{"schema":{"migration":0,"semantic":"MAJOR_0_MINOR_0_PATCH_1"},"id":"f4967818-890e-4d57-8071-cc6b78e965c4","name":"OS_SERVER","version":"10.1.15","platform":"LINUX","onboarded":true,"privacy":"OPEN","capabilities":"BLENDED","mechanism":"MANUAL","automaticUnload":false}]}


Thank you
Brendan

**Comment by hra42** [comment]
> [@hra42](https://github.com/hra42) if you are ever interested in moving closer w/ the Pieces team members internally, reach out to me via Email and we can have a chat. I appreciate you big time!

Thank you so much for your kind words. I send you an E-Mail!

**Comment by Oliver-Turp** [comment]
+1

**Comment by hassan-91120** [comment]
I have also sent mark an email from my gmail account hassanniazi697@gmail.com please let me know whether you have received it or not

**Comment by ismaildzf** [comment]
@mark-at-pieces 
I tried opening http://localhost:1000/applications and it didn't work for me. Is there another solution?

**Comment by mark-at-pieces** [comment]
hey @ismaildzf 

and @all:

We have release a new version of PiecesOS 11.0.0 (& the Desktop application) to target this issue as well as a few others.

It would be extremely helpful if yall how are still experiencing this issue retried to see if you can reproduce.

Update for this issue as well: If you are now signed in to Pieces you will have a greatly increased rate limit window.

On another note, you should now get a specific error message in the case of a rate limit within the copilot, so if you are still experiencing this issue please schedule some time w/ me.([link](https://calendar.app.google/knJSkkMhFomcQMcs7))

**Comment by mark-at-pieces** [comment]
also @ismaildzf w/ the update the port has adjusted from 1000 -> 39300, if you are still interested in sending me over the applications snapshot

**Comment by ellannjohnson** [comment]
I have the same issue. I tried the suggestions to visit http://localhost:1000/applications and send the results to Mark but the link doesn't work. Also, the thread said to message Mark on Discord but after reading through the server posts they ask us NOT to message them personally. I am not on the early access program.

**Comment by mark-at-pieces** [comment]
Anyone else experiencing this issue:

We are experiencing an outage with OpenAI specific models right now.

cc: https://github.com/pieces-app/support/issues/967#issuecomment-3608954628

cc: @ellannjohnson 

If anyone would like to hop on a call and debug/chat in real-time here is a link to our [support calendar](https://calendar.google.com/calendar/u/0/appointments/schedules/AcZssZ22WJ2Htd2wRMJhueCNYc0xbFBFCAN-khijcuoXACd_Uux3wIhgZeGkzDRcqD3teamAI-CwCHpr)🤘
